pyproject.toml

```diff
@@ -29,3 +29,49 @@ dependencies = [
 
 [tool.hatch.version]
 path = "src/smsfusion/__about__.py"
+
+[tool.hatch.envs.test]
+python = "3.10"
```

Comment:
Can we remove this line so that the tests are ran with the default Python? In the CI worflow we want to run the tests with different Python versions...

Reply:
Very good point